src/circuit/gadget/ecc/chip/witness_point.rs

```diff
@@ -70,7 +70,13 @@ impl Config {
         value: Option<(pallas::Base, pallas::Base)>,
         offset: usize,
         region: &mut Region<'_, pallas::Base>,
-    ) -> Result<(CellValue<pallas::Base>, CellValue<pallas::Base>), Error> {
+    ) -> Result<
+        (
+            AssignedCell<pallas::Base, pallas::Base>,
+            AssignedCell<pallas::Base, pallas::Base>,
+        ),
+        Error,
+    > {
```

Comment:
```suggestion
    ) -> Result<MagnitudeSign, Error> {
```

Reply:
This one is a point, not a magnitude-sign. Which makes your point :P